### DISCUSSION TITLE
Radio.garden Support

### DISCUSSION THREAD

**Discussion by YonakaDev** [web-scrobbler/web-scrobbler#4980] [Ideas]
I think this would be perfect since https://radio.garden plays music and its a very fun website as its a globe with radio stations all over the world. check it out :)

**Comment by sydniiiiiii** [comment]
[Triumph Trellis](https://triumphtrellis.com/product-category/trellis/) charming garden arch trellises add elegance and structure to any outdoor space, providing sturdy support for climbing plants to flourish and reach new heights. These beautifully designed arches create a captivating focal point, blending functionality with architectural charm as vines and blooms entwine. Perfect for enhancing the natural beauty of any garden, Triumph Trellis trellises transform ordinary spaces into lush, inviting sanctuaries.
